WFSServices/app/views/services/show.html.erb

```diff
@@ -65,15 +65,41 @@
   <%= @service.category %>
 </p>
 
+
 <p>
-  <strong>Contact:</strong>
-  <%= @service.contact %>
+  <strong>Contacts:</strong>
+
+  <% if @s_contacts.empty? %>
+    <p>No contacts at this time</p>
+  <% else %>
+    <ul>
+      <% @s_contacts.each do |i| %>
+        <li>
+          <%= i.first_name + ' ' + i.last_name %>
```

Comment:
For the part where you did "i.first_name + ' ' + i.last_name", it might be better to make a method in the contacts model called name or something, so that for the contacts index page, we can just call the method instead of duplicating code.